packages/metrics_center/lib/src/skiaperf.dart

```diff
@@ -317,16 +316,15 @@ class SkiaPerfGcsAdaptor {
   /// json files can be put in that leaf directory. We intend to use multiple
   /// json files in the future to scale up the system if too many writes are
   /// competing for the same json file.
-  static Future<String> computeObjectName(String githubRepo, String revision,
-      {GithubHelper githubHelper}) async {
+  static Future<String> computeObjectName(
+      String githubRepo, String revision, DateTime commitTime) async {
     assert(_githubRepoToGcsName[githubRepo] != null);
     final String topComponent = _githubRepoToGcsName[githubRepo];
-    final DateTime t = await (githubHelper ?? GithubHelper())
-        .getCommitDateTime(githubRepo, revision);
-    final String month = t.month.toString().padLeft(2, '0');
-    final String day = t.day.toString().padLeft(2, '0');
-    final String hour = t.hour.toString().padLeft(2, '0');
-    final String dateComponents = '${t.year}/$month/$day/$hour';
+    final DateTime commitTimeUtc = commitTime.toUtc();
```

Comment:
@keyonghan I switched the timestamps to be explicitly stored as UTC. Before they were based on the local time of the server it was run in. Is this okay?

Reply:
Is it easy to fix from test side rather than enforcing the UTC from code here?

Reply:
Okay switched back to previous functionality. This will cause issues if there are instances distributed globally running this logic.

Reply:
After revisiting the code, I agree that we need to enforce the timezone to be consistent.
But I guess our current logic is based on PST (could you please double check?). If so, let's enforce PST instead.

Reply:
For Cocoon, the GAE instance is in us-central. I presume the engine use case is running on LUCI, which has more variance.

I think UTC is the safest option to default to the data would only jump forward in time when this change occurs (unless there are instances in EMEA/Asia). SG?

Reply:
Just validated existing bucket is using UTC. Sounds good to me.